scripts/paycontract.py

```diff
@@ -0,0 +1,33 @@
+def pay(pay, balance1, balance2):
```

Comment:
Type hints won't forcibly typecast or error out but helps for reading the code
```suggestion
def pay(pay: int, balance1: int, balance2: int) -> [int, int]:
```